Automattic/abacus
# Experiment setup: Add copyable P2 post draft

Fixes https://github.com/Automattic/experimentation-platform/issues/950

#### Notes:
* The copy button may be repositioned, for example to the top-right where the "Edit" and "Assign metric" buttons are on the Overview tab.
* Target audience documentation is very simplistic (Yes/No). Added a comment in code.
* The exposure events are not used at the moment (not present in the P2 template).
* We're re-introducing private links even if they are not clickable. However, shortening is [not making much difference](https://github.com/Automattic/abacus/pull/802#issuecomment-1331746356).

#### Demo:
<img width="650" alt="Screenshot 2023-06-01 at 03 36 23" src="https://github.com/Automattic/abacus/assets/14192054/f67367d8-5eaf-4900-8b50-c652a5d187db">
<br><br>
<details>
<summary>Previous demos</summary>
<img width="600" alt="Screenshot 2023-05-31 at 23 00 01" src="https://github.com/Automattic/abacus/assets/14192054/6b87edf7-ba03-4ca0-a2cd-84b7ec279b0b">
<img width="600" alt="Screenshot 2023-05-27 at 04 30 09" src="https://github.com/Automattic/abacus/assets/14192054/1f095c53-ccad-4c7d-a813-41f378cffcfa">
</details>

#### To do:
- [ ] (post-merge) Update FG mentions from `code-setup` to `setup`.

## How has this been tested?






- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)
  - When testing locally, a Chrome flag should be used in order for the _Copy to clipboard_ action to work. See https://github.com/Automattic/abacus/pull/740#discussion_r908715048
